static/src/components/builder/components/ConditionalDisplay/conditionalDisplayTemplate.html

```diff
@@ -0,0 +1,3 @@
+<conditional-display id="{{id}}" :condition="{{{condition}}}">
```

Comment:
The `{{{ }}}` prevents html encoding, right? Will there be a problem if `condition` contains `"`?

Reply:
You are right. Im going to add a validation that checks if they use `"`,  to switch to `'`